lib/controller.dart

```diff
@@ -50,6 +56,11 @@ class AnalysisTabChangeData extends Data {
   AnalysisTabChangeData(this.tabIndex);
 }
 
+class ToggleOptionEnabledData extends Data {
+  bool isEnabled;
+  ToggleOptionEnabledData(this.isEnabled);
```

Comment:
nit:
```suggestion
  bool enabled;
  ToggleOptionEnabledData(this.enabled);
```
`enabled` is fairly suggestive in itself that its type is bool. Same for the other `is*Enabled` vars in this file, e.g. `_isDataComparisonEnabled` -> `_dataComparisonEnabled`

Reply:
ah yes makes sense, fixed